### PR TITLE
[Feature] Remove table property storage_cache_ttl

### PR DESCRIPTION
**Pull request by sduzh**
Fixes #issue

## What type of PR is this:
- [ ] BugFix
- [x] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Checklist:
- [ ] I have added test cases for my bug fix or my new feature
- [x] This pr will affect users' behaviors
- [x] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 6 / 6 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/lake/DataCacheInfo.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/ListPartitionInfo.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/RangePartitionInfo.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/common/util/PropertyAnalyzer.java|1|1|100.00%|[]|

File: fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java [hunk 1]
```diff
@@ -942,30 +941,13 @@ public static DataCacheInfo analyzeDataCacheInfo(Map<String, String> properties)
...
-        }
-
         boolean enableAsyncWriteBack =
                 analyzeBooleanProp(properties, PropertyAnalyzer.PROPERTIES_ENABLE_ASYNC_WRITE_BACK, false);
         if (!enableStorageCache && enableAsyncWriteBack) {
             throw new AnalysisException("enable_async_write_back can't be turned on when cache is disabled");
         }
 
-        return new DataCacheInfo(enableStorageCache, storageCacheTtlS, enableAsyncWriteBack);
+        return new DataCacheInfo(enableStorageCache, enableAsyncWriteBack);
```
[the remark this answers is not in the document]
Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java]: Fixed

File: fe/fe-core/src/main/java/com/starrocks/common/proc/PartitionsProcDir.java [hunk 2]
```diff
@@ -114,7 +114,6 @@ private void createTitleNames() {
                     .add("Buckets")
                     .add("DataSize")
                     .add("RowCount")
-                    .add("CacheTTL")
```
[the remark this answers is not in the document]
Comment (sduzh) [reply fe/fe-core/src/main/java/com/starrocks/common/proc/PartitionsProcDir.java:117]: Updated

**Review by wyb — APPROVED** [review]

**Review by gengjun-git — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 10 / 13 (76.92%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/common/proc/PartitionsProcDir.java|0|3|00.00%|[117, 410, 411]|
|:large_blue_circle:|com/starrocks/lake/LakeMaterializedView.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/lake/LakeTable.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/lake/DataCacheInfo.java|3|3|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/ListPartitionInfo.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/catalog/RangePartitionInfo.java|1|1|100.00%|[]|
|:large_blue_circle:|com/starrocks/common/util/PropertyAnalyzer.java|3|3|100.00%|[]|

**Comment by sduzh** [comment]
@Mergifyio backport branch-3.1

**Pull request merged**